1-js/08-prototypes/02-function-prototype/1-changing-prototype/task.md

```diff
@@ -1,12 +1,12 @@
-importance: 5
+اهمیت: 5
```

Comment:
You shouldn't translate Metadata. [Read more](https://github.com/javascript-tutorial/fa.javascript.info#metadata)